packages/edit-post/src/components/header/index.js

```diff
@@ -74,14 +76,25 @@ function Header() {
 						forceIsSaving={ isSaving }
 					/>
 				) }
-				<PreviewOptions
-					forceIsAutosaveable={ hasActiveMetaboxes }
-					forcePreviewLink={ isSaving ? null : undefined }
-				/>
-				<PostPreviewButton
```

Comment:
This is the old preview button and I couldn't see the use for it here since it seems to be hidden. It would be nice if someone with more experience with this code could check it out and make sure that this removal won't break something.

Reply:
Thanks @tellthemachines, I'll look into bringing it back.

Reply:
@tellthemachines I brought the old preview button back along with the required styles in dcbaf85d0da3b2d90cfce7110cd19e8c925fad3b.